gnat2goto/driver/tree_walk.adb

```diff
@@ -4704,6 +4707,32 @@ package body Tree_Walk is
                           Position => Global_Symbol_Table.Find (Expression_Id),
                           Process  => Set_Volatile'Access);
       end Handle_Pragma_Volatile;
+
+      procedure Handle_Pragma_Machine_Attribute (N : Node_Id) is
+         Argument_Associations : constant List_Id :=
+           Pragma_Argument_Associations (N);
+
+         --  first is the identifier to be given the attribute
+         First_Argument : constant Node_Id := First (Argument_Associations);
+
+         --  second is the attribute as string
+         Second_Argument : constant Node_Id := Next (First_Argument);
+         Attr_String_Id : constant String_Id :=
+           Strval (Expression (Second_Argument));
+         Attr_Length : constant Integer :=
+           Integer (String_Length (Attr_String_Id));
+      begin
+         String_To_Name_Buffer (Attr_String_Id);
+         if Name_Buffer (1 .. Attr_Length) = "signal" then
```

Comment:
Might be worth assigning this to a variable if you're using it multiple times (use `renames` if you're worried about making a copy)

Reply:
Renames works as well. Thanks.